chat_server/requirements.txt

```diff
@@ -0,0 +1,19 @@
+beautifulsoup4==4.11.1
+certifi==2022.9.24
+charset-normalizer==2.1.1
+click==8.1.3
+Flask==2.2.2
+Flask-Cors==3.0.10
+google==3.0.0
+gunicorn==20.1.0
+idna==3.4
+importlib-metadata==4.12.0
+itsdangerous==2.1.2
+Jinja2==3.1.2
+MarkupSafe==2.1.1
+requests==2.28.1
+six==1.16.0
+soupsieve==2.3.2.post1
+urllib3==1.26.12
+Werkzeug==2.2.2
+zipp==3.8.1
```

Comment:
nit: rasa-model uses env.yml

Reply:
(probably not worth changing though)

Reply:
Will add new issue for me to learn how to use conda